client/src/components/base/ItemLists/ItemImage/ItemImage.tsx

```diff
@@ -0,0 +1,43 @@
+import styled from '@emotion/styled';
+import React from 'react';
+
+import ItemWishButton from '../ItemWishButton/ItemWishButton';
+
+type ImageProps = {
+  quantity: number;
+};
+
+const ItemImage = ({ quantity }: ImageProps) => {
+  return (
+    <ImageContainer>
+      <ProductImg src="images/sampleItem.jpeg" />
+      {!quantity ? <SoldOutImg src="images/soldout.png" /> : ``}
+      {quantity ? <ItemWishButton /> : ``}
```

Comment:
```suggestion
      {quantity === 0 
      ? ( <SoldOutImg src="images/soldout.png" /> ) 
      : ( <ItemWishButton /> )}
```

이렇게 할 수도 있겠네요 (참고하세용)

Reply:
더 좋네요 👍 코드를 잘보시네요 ㅋㅋㅋㅋ